content/docs/code-splitting.md

```diff
@@ -45,7 +45,7 @@ console.log(add(16, 26)); // 42
 >
 > Your bundles will end up looking a lot different than this.
 
-If you're using [Create React App](https://github.com/facebookincubator/create-react-app), [Next.js](https://github.com/zeit/next.js/), [Gatsby](https://www.gatsbyjs.org/), or a similar tool, you will have a Webpack setup out of the box to bundle your
+If you're using [Create React App](https://github.com/facebook/create-react-app/), [Next.js](https://github.com/vercel/next.js/), [Gatsby](https://www.gatsbyjs.org/), or a similar tool, you will have a Webpack setup out of the box to bundle your
```

Comment:
Let's make this

https://create-react-app.dev/

and

https://nextjs.org/?

Reply:
Makes sense, done here https://github.com/reactjs/reactjs.org/pull/3123/commits/a0bb149dd59c066c02e9218ecb6eb86121a17612